src/ranged.cpp

```diff
@@ -693,7 +702,7 @@ dealt_projectile_attack player::throw_item( const tripoint &target, const item &
 // Draws the static portions of the targeting menu,
 // returns the number of lines used to draw instructions.
 static int draw_targeting_window( WINDOW *w_target, item *relevant, player &p, target_mode mode,
-                                  input_context &ctxt )
+                                  input_context &ctxt, std::vector<aim_type> *aim_types )
```

Comment:
It cannot be null. It should work fine passing a const reference instead, and I can see that would be better practice. I'll make a new PR to amend.